spec/system/candidate_interface/course_selection/candidate_attempts_to_add_course_from_find_to_application_from_previous_cycle_spec.rb

```diff
@@ -1,51 +0,0 @@
-require 'rails_helper'
-
-RSpec.feature 'Candidate attempts to add course via Find to application from previous cycle' do
```

Comment:
Other deletions look good — is it worth hanging on to this one, though?

Reply:
mmm ... yes actually we should keep that one

Reply:
Restored